src/resampling.jl

```diff
@@ -113,16 +124,129 @@ function train_test_pairs(cv::CV, rows)
     # define the (trainrows, testrows) pairs:
     firsts = 1:k:((nfolds - 1)*k + 1) # itr of first `test` rows index
     seconds = k:k:(nfolds*k)          # itr of last  `test` rows index
+
     ret = map(1:nfolds) do k
         f = firsts[k]
         s = seconds[k]
+        k < nfolds || (s = n_observations)
         return (vcat(rows[1:(f - 1)], rows[(s + 1):end]), # trainrows
-                rows[f:s])                               # testrows
+                rows[f:s])                                # testrows
     end
 
     return ret
 end
 
+"""
+    stratified_cv = StratifiedCV(; nfolds=6,  shuffle=false, rng=Random.GLOBAL_RNG)
+
+Stratified cross-validation resampling strategy, for use in
+`evaluate!`, `evaluate` and in tuning. Applies only to classification
+problems (`OrderedFactor` or `Multiclass` targets).
+
+    train_test_pairs(stratified_cv, rows, X, y)        # X is ignored
```

Comment:
since `X` is ignored could we not have a 3 argument call? (`scv, rows, y`) ?

Reply:
Thanks for the review. 

Re partition. Sounds like good idea. Since `partition` is currently in `MLJBase` let's make this a separate PR. Raised issue [here](https://github.com/alan-turing-institute/MLJBase.jl/issues/113)

Reply:
No, we can have a 3 argument call. Will implement